File: application/src/main/java/ch/pristane/laverie/smart/application/dtos/BookingDto.java
package ch.pristane.laverie.smart.application.dtos;

import ch.pristane.laverie.smart.domain.enums.BookingStatus;

import java.time.OffsetDateTime;
import java.util.UUID;

public record BookingDto(
        UUID id,
        UUID machineId,
        OffsetDateTime startTime,
        OffsetDateTime endTime,
        String customerName,
        BookingStatus status
) {
}

File: src/PristaneLaverieSmart.Application/Features/Bookings/Dtos/BookingDto.cs
using PristaneLaverieSmart.Domain.Enums;

namespace PristaneLaverieSmart.Application.Features.Dtos;

public sealed record BookingDto (
    Guid Id,
    Guid MachineId,
    DateTimeOffset StartTime,
    DateTimeOffset EndTime,
    string CustomerName,
    BookingStatus Status
);